README.md

```diff
@@ -291,158 +334,170 @@ date: {
-    ],
-  },
-],
-```
-This block configures the table of contents plugin, it specifies the nodeHierarchy that the node has to follow. At the moment, we ask not to change this configuration while we iron out the kinks in the public interface for this library. Essentially this relates to the [prosemirror schema](https://prosemirror.net/docs/guide/#schema), which is not yet configurable in this package.
-
-```javascript
-articleStructure: {
-  mode: 'besluit',
-}
-```
-This block configures the article-structure plugin. The article-structure plugin has 2 modes:
+- `placeholder`:
+	- `insertDate`: placeholder to show when inserting a date that has no data yet.
+	- `insertDateTime`: placeholder to show when inserting a datetime that has no data yet.
+- `formats`: specify default formats to show for selection in the date card.
+	- `label` (optional): The label shown to the user on the card. If not provided, the format is used instead e.g.: `dd/MMvariable/yyyy`
+	- `key`: A **unique** identifier used for identification in the internal code. 
+	- `dateFormat`: The date format used when this is selected.
+	- `dateTimeFormat`: The datetime format to use when this is selected. Used when the user selected "Include time".
+- `allowCustomFormat`: true/false, determines if the option to insert a fully custom format is available.
+
+For more information about the syntax of defining date(time) formats check the documentation of the underlying library used [date-fns](https://date-fns.org/v2.29.3/docs/format).
 
-- 'besluit'`: for manipulating articles in decisions
-- 'regulatoryStatement'`: for manipulating chapters, sections, etc in regulatory statements.
+### Roadsign Regulation
+Add annnotated *mobiliteitsmaatregelen* from a specified registry, which will most likely be using the [public facing sparql endpoint](https://register.mobiliteit.vlaanderen.be/sparql) of [the roadsign registry](https://register.mobiliteit.vlaanderen.be). This data is maintained by experts at [MOW Vlaanderen](https://www.vlaanderen.be/departement-mobiliteit-en-openbare-werken).
 
-```javascript
-roadsignRegulation: {
-  endpoint: 'https://dev.roadsigns.lblod.info/sparql',
-  imageBaseUrl: 'https://register.mobiliteit.vlaanderen.be/',
-}
-```
-This block configures the roadsign-regulation plugin.
-The plugin exposes 2 variables to configure, the first one specifies the endpoint to fetch the roadsigns, and the second one is a way to fix the images that don't specify a base url, in this case `https://register.mobiliteit.vlaanderen.be/` will be used.
+:warning: This plugin will only activate in *besluiten* with a certain rdf type.
+<details><summary>Exhaustive list of types</summary>
 
-### Enabling/disabling the environment banner
-The environment banner is a visual indication of the environment you are currently using and which versions of embeddable, the editor and editor-plugins are in use.
+`https://data.vlaanderen.be/id/concept/BesluitType/4d8f678a-6fa4-4d5f-a2a1-80974e43bf34`
```

Comment:
Maybe the types in this list should be converted to links?

Reply:
Right, I didn't think the links would actually go to something (as they don't always...)
but in this case they do, so links make more sense.